mon-pix/app/services/storage.js

```diff
@@ -10,4 +10,8 @@ export default class Storage extends Service {
   getLogin() {
     return sessionStorage.getItem(SESSIONSTORAGE_LOGIN);
   }
+
+  clearLogin() {
+    return sessionStorage.removeItem(SESSIONSTORAGE_LOGIN);
+  }
```

Comment:
Il me semble qu'un vidage complet du `sessionStorage` cf. https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage serait plus adéquate car plus générique, en effet dans un `beforeEach`/`afterEach` on veut que tout soit propre et dans un état initial mais on ne sait pas que tel autre test a rempli des informations de login ou autres : 

```suggestion
  clear() {
    return sessionStorage.clear();
  }
```